contribs/gmf/src/services/permalink.js

```diff
@@ -700,8 +707,16 @@ gmf.Permalink.prototype.registerMap_ = function(map, oeFeature) {
   //     b) the X, Y and Z available within the permalink service, if available
   if (oeFeature && oeFeature.getGeometry()) {
     const size = map.getSize();
+    const geom = oeFeature.getGeometry();
```

Comment:
oeFeature.getGeometry() is already called two line upper. Can you call it once (use the variable) ?